server/src/main/java/org/bonitasoft/console/common/server/i18n/I18n.java

```diff
@@ -43,45 +57,86 @@ public static I18n getInstance() {
         return (I18n) I18N_instance;
     }
 
-    private static File getI18nDirectory() {
-        String bonitaHome = System.getProperty("bonita.home");
-        String i18nPath = "client" + File.separator + "platform" + File.separator + "work" + File.separator + "i18n"
-                + File.separator;
-        return new File(bonitaHome, i18nPath);
+    // For test matters only:
+    void refresh() {
+        I18N_CUSTOM_DIR = getI18nCustomDirectory();
+    }
+
+    private static File getI18nCustomDirectory() {
+        String customI18nFolder = System.getProperty(I18N_CUSTOM_DIR_PROPERTY);
+        return customI18nFolder != null ? new File(customI18nFolder) : null;
     }
 
     @Override
     public void loadLocale(final LOCALE locale) {
-        Map<String, String> results = loadLocale(locale, FileUtils.listDir(I18N_DIR));
+        Map<String, String> results = null;
+        if (I18N_CUSTOM_DIR != null) {
+            results = loadLocale(locale, FileUtils.listDir(I18N_CUSTOM_DIR));
+        }
+        if (results == null || results.isEmpty()) {
```

Comment:
seen with anthony, it would be better to overwrite keys in classpath be keys in custom dir, this code is not working like this. can you find if it is possible?